frontend/src/pages/Scans/ScansView.tsx

```diff
@@ -179,6 +181,8 @@ const ScansView: React.FC = () => {
       setOrganizationOptions(
         organizations.map((e) => ({ label: e.name, value: e.id }))
       );
+      const tags = await apiGet<OrganizationTag[]>(`/organizations/tags`);
```

Comment:
I think we should move this call up before `setScans`, so that all the data is populated only after all the fetch requests have completed.